tests/processing/test_process_queue.py

```
@@ -0,0 +1,42 @@
+import unittest
```

Ruff

`unittest` imported but unused

Remove unused import: `unittest`

(F401)